Makefile

```diff
@@ -1,3 +1,8 @@
+.PHONY: help envvar \
+				check_links check_spelling \
```

Comment:
Not sure why these look like they're very far off to the right, but in `emacs` they're lined up properly...

Reply:
Probably due to GitHub's tab formating - it shows up as 8 spaces instead of 4.

Reply:
ah OK if it doesn't show up for anyone in the actual code like that maybe it's fine? If it's only a GitHub issue then I won't worry about trying to fix it.

Reply:
I guess it will show up in everybody's IDE based on their tab configuration, that's why tabs are evil :)

Reply:
oh I could have sworn I had spaces set -- but you're absolutely right, I tabbed these, over, will fix!

Reply:
OK I think I've gotten most of the spacing discrepancies after turning whitespace mode on. I tried to just remove all the tabs from the file at first but then that triggered changes on just about every line, so went through and tried to match up the places that were tabs and the new code I added that was unfortunately tabbed with a different width.